examples/lightgbm/distributed-training.ipynb

```diff
@@ -81,7 +81,11 @@
    },
    "outputs": [],
    "source": [
-    "lightgbm.execute(config=params, docker_registry=DOCKER_REGISTRY)"
+    "lightgbm.execute(config=params,\n",
+    "                          docker_registry=DOCKER_REGISTRY,\n",
+    "                          cores_per_worker=1,\n",
+    "                          memory_per_worker=0.5,\n",
```

Comment:
similarly, is this 1/2 the ram of the node, or 0.5GB? possibly adding a comment at the end of the line might clarify the usage. something like:
 memory_per_worker=0.5,\n",   # reserve 0.5GB for each worker

Reply:
0.5gb . Added doc strings.